utils/mil_msgs/msg/LabeledImage.msg

```diff
@@ -0,0 +1,7 @@
+# Represents the complete data from a labeled image from LabelMe
+
+sensor_msgs/Image image            # Image where objects are labeled
```

Comment:
But when you play back to test don't you want to be able to have all original data at your disposal? Particularly Odom and anything else in the bag.